package.json

```diff
@@ -18,22 +18,24 @@
     "lint": "npm run lint --if-present -ws && prettier -c .",
     "version": "changeset version && npm i",
     "release": "changeset publish",
-    "storybook": "npm run build && npm run dev -w @holisticon/hap-documentation"
+    "storybook": "npm run build && npm run dev -w @holisticon/hap-documentation",
+    "update": "ncu -i -ws"
```

Comment:
Anstatt manuelle Update checks zu machen, wäre ich dafür Dependabot einzuschalten. Sobald wir visual regression testing am start haben, könnten wir minor/patch updates dann sogar automatisch mergen lassen.

Reply:
Persönlich bin ich kein Fan von Dependabot. Dadurch vernachlässigen Projekte häufig die Möglichkeit mit Dependency Updates auch zu modernisieren und neue Features wahrzunehmen. Bei bewussten Updates habe ich die Change kleinere Neuerungen gleich mit anzuwenden. Und soweit ich weiß aktualisiert Dependabot nicht die Node.js Version, oder?

Ich investiere da lieber einmal alle 1-2 Wochen 15 Minuten meiner Zeit und mach ein schnelles Update und gucke bei interessanten Dependencies einmal in die Changelogs.

Reply:
Valider Punkt. Ich würde erstmal auch kein Automerge einschalten. Wenn uns Dependabot nur die PRs für neue Versionen erstellt, sollten wir entsprechend einmal kurz überlegen, ob das eine inhaltlich spannende Änderung sein könnte.